From 61b3fb07f48c6edab2c3c9b516f5a7518f804b99 Mon Sep 17 00:00:00 2001
From: Joone Hur <joone.hur@intel.com>
Date: Mon, 24 Aug 2015 19:58:34 -0700
Subject: [PATCH] Do not include remoting/remoting.gyp:remoting_perftest

Running GYP failed due to remoting.gyp:remoting_perftest for ozone.

https://codereview.chromium.org/1228283004
---
 remoting/remoting_all.gyp | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/remoting/remoting_all.gyp b/remoting/remoting_all.gyp
index b624351..de068c3 100644
--- a/remoting/remoting_all.gyp
+++ b/remoting/remoting_all.gyp
@@ -19,7 +19,6 @@
         '../remoting/remoting.gyp:remoting_breakpad',
         '../remoting/remoting.gyp:remoting_browser_test_resources',
         '../remoting/remoting.gyp:remoting_client',
-        '../remoting/remoting.gyp:remoting_perftests',
         '../remoting/remoting.gyp:remoting_protocol',
         '../remoting/remoting.gyp:remoting_resources',
         '../remoting/remoting.gyp:remoting_test_support',
@@ -52,6 +51,11 @@
             '../remoting/remoting.gyp:remoting_unittests_apk',
           ],
         }],
+        ['use_ozone==0', {
+          'dependencies': [
+            '../remoting/remoting.gyp:remoting_perftests',
+          ],
+        }],
         ['enable_remoting_host==1', {
           'dependencies': [
             '../remoting/remoting.gyp:remoting_infoplist_strings',
-- 
1.9.1

